### ISSUE TITLE
Can't init the dlite beta

### ISSUE THREAD

**Issue by synic** [nlf/dlite#209]
## Bug Reports
- dlite version in use (run `dlite --version`): 

dlite version 2.0.0-beta8
OS X 10.12.1 (16B2657)
- expected behavior:

It to start dlite
- actual behavior:

```
[Projects]$ dlite init

Virtual machine hostname [local.docker]:
Disk size (in gigabytes) [20]:
CPU cores to allocate to VM [2]:
Memory to allocate to VM (in gigabytes) [2]:
exit status 1
```

**Comment by nlf** [comment]
what result do you get if you run the command

``` sh
defaults read /Library/Preferences/SystemConfiguration/com.apple.vmnet.plist Shared_Net_Address
```

**Comment by synic** [comment]
```
```

**Comment by synic** [comment]
```
[~]$ defaults read /Library/Preferences/SystemConfiguration/com.apple.vmnet.plist Shared_Net_Address

2016-10-30 11:14:21.094 defaults[47286:709452]
The domain/default pair of (/Library/Preferences/SystemConfiguration/com.apple.vmnet.plist, Shared_Net_Address) does not exist
```

**Comment by nlf** [comment]
that's interesting.. does that file exist? can you paste the contents here?

**Comment by synic** [comment]
It does not exist

**Comment by nlf** [comment]
what version of OS X are you on? have you ever successfully used a previous version of dlite? or xhyve? docker for mac?

**Comment by synic** [comment]
I'm on Sierra, a fresh install.  I've used dlite before, but not on a fresh Sierra install.

Docker for Mac works

edit:  I have also used dlite on a non-fresh sierra install.  Maybe it's not a file present in Serria but just happened to be there because I upgraded?

**Comment by nlf** [comment]
it looks like the file may only exist if you've already run a vm that uses vmnet before, so this is kind of a chicken and egg problem. i'm probably going to add a fallback and assume a default so the init process can continue and if it turns out to be wrong i'll have to figure out a more robust solution

**Comment by synic** [comment]
Is there a way I can cause the file to be created so I can use the prerelease?

**Comment by nlf** [comment]
sorry for the delay! i'll release another beta build today or tomorrow that doesn't need the file to exist. i added a fallback so it will use a default

**Comment by bartrail** [comment]
I'm running also into this issue on 2.0.0-beta8 (same output on `defaults read /Library/Preferences/SystemConfiguration/com.apple.vmnet.plist Shared_Net_Address`) - is there any way to get past this error yet without building from sources? or could you release another beta build please?

**Comment by nlf** [comment]
just released 2.0.0-beta9 that has a fallback in place for this issue

**Comment by okenwa103** [comment]
Just release 2.0.0

**Comment by okenwa103** [comment]
just released 2.0.0-beta9 that has a fallback in place for this issue
